model/riscv_types_kext.sail

```diff
@@ -68,8 +68,15 @@ function aes_mixcolumn_inv(x) = {
   b3 @ b2 @ b1 @ b0 /* Return value */
 }
 
-val aes_decode_rcon : bits(4) -> bits(32)
+/* Turn a round number into a round constant for AES. Note that the
+   AES64KS1I instruction is defined such that the r argument is always
+   in the range 0x0..0xB. Values of rnum outside the range 0x0..0xA
+   do not decode to the AES64KS1I instruction. The 0xA => 0x0 case is used
```

Comment:
> The 0xA => 0x0 case is used

There is no 0xA => 0x0 case, the function is just never called and no XOR is ever performed.

Reply:
(i.e. just drop the => 0 and I think the sentence reads fine?)

Reply:
Agreed, just fixed this now.